### PR TITLE
Update reboot docs

### LINKED ISSUE #2108: outdated and dangerous advice for reboots
From troubleshooting.md#reboots

> because the process of getting everything re-started and re-connected via restart policies is not entirely reliable, the router and proxy containers do not have Docker restart policies set. Until that changes, we recommend that you create appropriate startup scripts to launch Weave Net and run application containers from [your favourite process manager](/site/systemd.md).
>
> If you are shutting down or restarting a host deliberately, run `weave reset` to clear everything down.
> 
> The Weave Net Docker plugin does restart automatically because it must always start with Docker, as described in [its documentation](/site/weave-docker-api/using-proxy.md).

This is all complete nonsense now that we a) *do* configure restart policies for everything, and b) have permanent peer IDs and persistence (so you most certainly do not want to run `weave reset` on restarts.)

### PR DESCRIPTION
**Pull request by bboreham**
Fixes #2108

### CODE REVIEW

File: site/troubleshooting.md [hunk 1]
```diff
@@ -275,19 +275,13 @@ re-connect.
...
+Docker restart policy `--restart=always`, so they will come back again
+on reboot.
 
-If you are shutting down or restarting a host deliberately, run `weave
-reset` to clear everything down.
-
-The Weave Net Docker plugin does restart automatically because it must
-always start with Docker, as described in
-[its documentation](/site/weave-docker-api/using-proxy.md).
+Note that `weaveproxy` is responsible for re-attaching everything at
```
Comment (rade) [inline site/troubleshooting.md]: "...responsible for reconfiguring the Weave router and re-attaching application containers to the Weave network".

File: site/troubleshooting.md [hunk 2]
```diff
@@ -275,19 +275,13 @@ re-connect.
...
+startup, so if you choose not to run `weaveproxy` you must make other
+arrangements.
```
Comment (rade) [inline site/troubleshooting.md]: such as?

**Comment by rade** [comment]
@bboreham please take a quick look at my commit. If you are happy I will do some squashing and merge.

**Comment by bboreham** [comment]
LGTM

**Pull request merged**